examples/notebooks/segmentation-tutorial.ipynb

```diff
@@ -57,7 +57,7 @@
    "outputs": [],
    "source": [
     "# Catalyst\n",
-    "!pip install catalyst==20.10.1\n",
+    "!pip install catalyst\n",
```

Comment:
in my opinion, it is better to freeze the version

Reply:
Yes, I agree with you, but I don't know what version I should to write